src/commands/createFunction.ts

```diff
@@ -33,22 +33,17 @@ const requiredFunctionAppFiles: string[] = [
     path.join('.vscode', 'launch.json') // NOTE: tasks.json is not required if the user prefers to run 'func host start' from the command line
 ];
 
-function validateTemplateName(rootPath: string, name: string | undefined, language: string): string | undefined {
+function validateTemplateName(rootPath: string, name: string | undefined): string | undefined {
     if (!name) {
         return localize('azFunc.emptyTemplateNameError', 'The template name cannot be empty.');
     }
-
-    if (language === ProjectLanguage.Java) {
-        return validateJavaFunctionName(name);
-    } else {
-        if (fse.existsSync(path.join(rootPath, name))) {
-            return localize('azFunc.existingFolderError', 'A folder with the name \'{0}\' already exists.', name);
-        }
-        if (!functionNameRegex.test(name)) {
-            return localize('azFunc.functionNameInvalidError', 'Function name must start with a letter and can contain letters, digits, \'_\' and \'-\'');
-        }
-        return undefined;
+    if (fse.existsSync(path.join(rootPath, name))) {
```

Comment:
This case still does not apply for Java, correct?

Reply:
(I'm guessing you should be using `getNewJavaFunctionFilePath` to test the path's existence rather than `rootPath`)